.github/workflows/test.yml

```diff
@@ -10,9 +10,15 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: '12.x'
+      - name: Start MongoDB
+        uses: supercharge/mongodb-github-action@1.3.0
+        with:
+          mongodb-version: '4.0'
       - name: Install deps
         run: yarn install
       - name: Build everything
         run: yarn build:production
       - name: Run tests
         run: yarn test
+        env:
+          TEST_MONGO_URL: 'mongodb://localhost:27017/wepublish_test'
```

Comment:
Suggesting to move the up, befoer `steps:`

(btw why Test: is starting with capitals above?)

Reply:
... further i am unsure if this is correct here, should this not be in the docker file at `/api`?

Reply:
@dev7ch this is how github-actions work. And no the `TEST_MONGO_URL` is not needed in the docker-compose.yaml since that file is not need anywhere but local dev environment.

Reply:
@roosnic1 yes, thats correct standard way so set env in the jobs (as they are independent), maybe i was not clear enough, my idea/sugesstion was to set the variable on the workflow level, if possible, instead of job level to reuse the var in upcoming tests:

- For readability
- For having only one to define

https://github.community/t/how-to-set-and-access-a-workflow-variable/17335

Reply:
Good point but so far we only need the `TEST_MONGO_URL` var for the tests with mongodb. And the `yarn test` command starts all the tests anyway. But this can/will change as the projects grows. As an old friend of mine always said "Premature optimization is the death of any project" 😄.

Reply:
😄